Doc/library/asyncio.rst

```diff
@@ -42,7 +42,11 @@ programming: see the :ref:`Develop with asyncio <asyncio-dev>` page which lists
 common traps and explains how to avoid them. :ref:`Enable the debug mode
 <asyncio-debug-mode>` during development to detect common issues.
 
-High-level APIs:
+
+Contents
+========
```

Comment:
Probably should drop this down a level since it creates a TOC entry Contents

Reply:
<img width="686" alt="screenshot 2018-09-14 14 36 57" src="https://user-images.githubusercontent.com/2680980/45576372-bb489200-b82b-11e8-985d-d80c7155311a.png">

Reply:
Good catch!